setup.cfg

```diff
@@ -22,10 +22,11 @@ classifiers =
     Programming Language :: Python :: 3.9
     Programming Language :: Python :: 3.10
     Programming Language :: Python :: 3.11
+    Programming Language :: Python :: 3.12
 
 [options]
 zip_safe = False
-python_requires = >=3.9,<3.12
```

Comment:
I think I'd rather keep this pinned to <3.12 for now, until we put together our DMS B10.1 release in a few weeks and do thorough testing of that whole release under 3.12.

Reply:
Sounds good. I think that will mean the CI won't be able to test 3.12 so I'll pull those changes out of this PR.